### PR TITLE
wasmcloud-actor-derive -> wasmcloud-actor-macros

### LINKED ISSUE #392: Rename `wasmcloud-actor-derive` -> `wasmcloud-actor-macros`
In alignment with #388

### PR DESCRIPTION
**Pull request by RoniDey1819**
## Feature or Problem

Renamed wasmcloud-actor-derive -> wasmcloud-actor-macros

## Related Issues

Closes #392

## Testing

Built on platform(s)
- [ ] x86_64-linux
- [ ] aarch64-linux
- [ ] x86_64-darwin
- [ ] aarch64-darwin
- [ ] x86_64-windows

Tested on platform(s)
- [ ] x86_64-linux
- [ ] aarch64-linux
- [ ] x86_64-darwin
- [ ] aarch64-darwin
- [ ] x86_64-windows

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -134,7 +134,7 @@ uuid = { version = "1", default-features = false }
 wascap = { version = "0.11", default-features = false }
 wasi-common = { version = "13", default-features = false }
 wasmcloud-actor = { version = "*", path = "./crates/actor" }
-wasmcloud-actor-derive = { version = "*", path = "./crates/actor/derive" }
+wasmcloud-actor-macros = { version = "*", path = "./crates/actor/derive" }
```
Comment (rvolosatovs) [inline Cargo.toml:137]: Could you also move the directory to `crates/actor/macros` ?

**Review by rvolosatovs — DISMISSED** [review]
Thank you for contribution!
I am happy to merge as-is, but would be nice to move the crate to appropriate location and please make sure to sign-off your commits, so we can merge

File: Cargo.toml (hunk 1 above)
Comment (RoniDey1819) [reply Cargo.toml:137]: @rvolosatovs check this out

**Review by rvolosatovs — APPROVED** [review]

**Comment by rvolosatovs** [comment]
Hey @RoniDey1819, I just rebased your branch, fixed up a little issue still left with moving of the directory and signed-off the commit to comply with https://github.com/apps/dco, hope you don't mind.

I think this is good to merge, thank you!

**Pull request merged**